tomcat9-maven-plugin/src/main/java/org/openmrs/maven/plugins/RunTomcat.java

```diff
@@ -113,7 +113,7 @@ public void execute() throws MojoExecutionException {
 						"Please note that it is not recommended to run OpenMRS platform " + server.getPlatformVersion()
 								+ " on JDK 8.\n");
 			}
-		} else if (platformVersion.getMajorVersion() == 2) {
+		} else if (platformVersion.getMajorVersion() == 2 || platformVersion.getMajorVersion() == 3) {
```

Comment:
Will you again modify this when we get to platform 4, 5, 6, etc?

Reply:
fixed